janusgraph-examples/example-tinkergraph/conf/jgex-tinkergraph.properties

```diff
@@ -0,0 +1,6 @@
+gremlin.graph=org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerGraph
+
+// DefaultIdManager types: INTEGER, LONG, UUID, ANY
```

Comment:
The comment characters in a properties file is `#`, not `//`:

https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html

Incidentally, `!` appears to also be recognized as a comment character, but I would suggest sticking with the more standard `#` since that's more widely used in many other file formats.